sdk/appconfiguration/app-configuration/package.json

```diff
@@ -77,11 +77,11 @@
     "@azure/core-http": "^1.0.0",
     "@azure/core-paging": "^1.1.0",
     "@azure/core-tracing": "1.0.0-preview.7",
-    "@azure/identity": "^1.0.0",
     "@opentelemetry/types": "^0.2.0",
     "tslib": "^1.10.0"
   },
   "devDependencies": {
+    "@azure/identity": "^1.1.0-preview1",
```

Comment:
I'm a little late here, but I think the original plan was to never have `@azure/identity` be a dependency of an SDK library.  Should this be in `peerDependencies` instead if the goal is to ensure a min version of Identity?

/cc @schaabs @bterlson

Reply:
peerDependencies are painful, so I'd prefer to avoid them. I think a devDependency is fine if it's only used for testing purposes, it won't affect consumers.